solutions/src/main/java/ru/sberbank/school/task07/Test.java

```diff
@@ -0,0 +1,47 @@
+package ru.sberbank.school.task07;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Map;
+
+public class Test {
+    public static void main(String[] args) {
+        String path = "C:\\Users\\Anastasia\\Desktop\\Java\\Tasks\\homework\\solutions\\src\\main\\java\\" +
```

Comment:
&#39;+&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1972737698>*